client/src/components/JurisdictionAdmin/DownloadBatchTallySheetsButton/DownloadBatchTallySheetsButton.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from 'react'
+import { AnchorButton } from '@blueprintjs/core'
+import { BlobProvider, Document } from '@react-pdf/renderer'
+import { toast } from 'react-toastify'
+
+import BatchTallySheet from './BatchTallySheet'
+import useContestsJurisdictionAdmin from '../useContestsJurisdictionAdmin'
+import { useBatches } from '../useBatchResults'
+
+const AnchorButtonErrorState = (): JSX.Element => {
+  useEffect(() => {
+    toast.error('Error preparing batch tally sheets for download')
+  }, [])
```

Comment:
It may also make sense to log this error to Sentry. I know client-side Sentry automatically reports unhandled client-side errors. This case is interesting, though, because the error is handled, so we can't rely on that behavior. We could explicitly report to Sentry here

Reply:
Hmm yeah failure here would be unexpected and we'd probably want to know. I like the idea of explicitly reporting

Reply:
https://github.com/votingworks/arlo/pull/1475/commits/34871cae983dfeede2215d717d06636a6377f1bf